source/module_elecstate/module_charge/charge_mixing.cpp

```diff
@@ -796,6 +796,13 @@ void Charge_Mixing::mix_rho_real(Charge* chr)
 
 void Charge_Mixing::mix_dmr(elecstate::DensityMatrix<double, double>* DM)
 {
+    if (GlobalV::MIXING_RESTART <= 0 
```

Comment:
I have found two similar parameter: Charge_Mixing::mixing_restart and GlobalV::MIXING_RESTART .
In principle, we should avoid to use global variables in code, what is difference between these two variables?

Reply:
I rename `mixing_restart` as `mixing_restart_step` to avoid misunderstaning